pgjdbc/src/test/resources/logging.properties

```diff
@@ -0,0 +1,4 @@
+.level = INFO
+handlers = java.util.logging.ConsoleHandler, java.util.logging.FileHandler
```

Comment:
I debated whether or not to include it. There is a server file logged during testing (`pgjdbc/target/pgjdbc-tests.log`), so we can control the format of that file as well from this properties file. Easy enough to remove if you'd prefer, and easy enough to change at a latter date.